src/notation/view/noteinputbarmodel.cpp

```diff
@@ -122,6 +111,18 @@ QHash<int, QByteArray> NoteInputBarModel::roleNames() const
 
 void NoteInputBarModel::load()
 {
+    uiConfiguration()->toolConfigChanged(TOOLBAR_NAME).onNotify(this, [this]() {
+        load();
```

Comment:
When `load` is called again, the `onNotify`s will also be called again... is that problematic? Maybe split the method into `load` and `reload`, where `load` calls `reload`?

Reply:
by default do not subscribe a second time